packages/fx-core/src/component/driver/botFramework/createOrUpdateBot.ts

```diff
@@ -77,6 +79,10 @@ export class CreateOrUpdateBotFrameworkBotDriver implements StepDriver {
     try {
       this.validateArgs(args);
 
+      if (!isUUID(args.botId)) {
```

Comment:
Any concern if put this into `validateArgs`?

Reply:
Just let `validateArgs` only validate against yml schema.